### ISSUE TITLE
Invalid signup link

### ISSUE THREAD

**Issue by MrDeKat** [steemit/condenser#1597]
My first attempt at signing up was on the 21st of june. I got an email address after completing the signup completely. I clicked the email and it said that it was invalid.

Great news, we've just approved your account.
 
You now have exclusive control over your Steem account, plus some free Steem to help you get started.
 
On Steemit you get rewarded for your curation and content according to the will of the crowd. To login, click the link below.
 
https://steemit.com/start/2de0f1d2f3771114104037c4dd 
----------------------------------------------

"This is not a valid sign up code. Please click the link in your welcome email."

-----------------------------------------------

So i tried to sign up again. Used a different user name, email and didnt get the page to verify my phone number... my second one. After that, tried again... username taken, email taken. I didnt even finish signing up... So thats 2 emails two usernames and 1 phone number eaten by the internet trolls. 

Tried again... put in a username, put in email... deleted email and backspaced because i wasnt happy with the user name.... but decided it was ok... username taken. 3 user names 2 emails and 1 phone number eaten by teh internet trolls.

Tried again... put in random junk username because ive used all my internet aliases at this point... used another of my email addresses... put in my wife phone number and... nothing. I havent gotten an email or anything.

That has been about 3 weeks ago. Soooo.... Im out of phones. Im out of email addresses and im out of unique to me usernames. Thanks? I would really like to use your platform but you guys are making me not really wanna deal with it. This is my last ditch effort to get in touch. I have tried Facebook, twitter and when i checked your slack as screwed up. No responses to any of them. A patron commented on the facebook one and said that the next update should fix things like htis... but that doesnt help me if i have already used all my means of signing up... It would be great if you could delete my attemps or something and let me try again. Because as of right now, its is bs and i cant advocate anyone else to use even try signing up...

**Comment by originated** [comment]
@MrDeKat Once a sign up code(link) is used, it will no longer be valid.  In regards to your usernames once they have been reserved for a account they are no longer available.  Receiving sign up codes  via email can take time due to a high demand for signups. Please contact me directly on steemit chat at "originate" to assist clearing your attempts/which should resolve your issue.

**Comment by sneak** [comment]
I'm going to use this issue to track the main bug reported, which is not about signing up multiple times, but that the signup link that was emailed didn't work.

@valzav - can you reproduce this behavior?

**Comment by plink01001** [comment]
Have there been any recent reports of this, or can it be closed?

**Comment by MrDeKat** [comment]
Except that i still havent gotten an email for anything but the first one... that didnt work... Nothing from STEEMIT at all. Its only been about 4 months, if thats an acceptable timeframe to get a verification email... That and i still havent gotten a new phone number or anyone to let me use their phone to sign up since then. So i have no clue if its still an issue or not.

Kind of ridiculous that this has happened.

**Comment by TimCliff** [comment]
@MeDeKat - can you ask for help in the help channel of Steemit chat?

**Comment by plink01001** [comment]
There has been one new report of this within the past two weeks.

**Comment by TimCliff** [comment]
Several users have been reporting instances of this recently. I will provide details privately.

**Comment by MrDeKat** [comment]
About time... Its only been about 6 months for me. Would have loved to have been some of the first people, but half a year in tech/social media... is an eternity. Too bad i missed its growing up. I guess its still going to grow but... sigh... I tried... and tried... and tried... and tried... and tried... and spammed facebook, twitter... posted on here. Nobody got with me. Its pretty bs if you ask me. Wheres the customer support? So anyway, yeah. Im out of ways to sign up and been out of ways to sign up... Used 3 different phones and 3 different emails with 3 different versions of my unique names... all accepted, but never given a chance to actually finalize the setup, so they are all taken now....

Hope you email me or something and figure out how to get me in finally...

**Comment by TimCliff** [comment]
@MrDeKat - the GitHub issue is to track the bug with the website, but is not for troubleshooting your individual account issue. As stated above, and in the repository's <a href="https://github.com/steemit/condenser/blob/master/CONTRIBUTING.md">Guidelines for contributing</a>, if you need help with your signup - please ask for help in the #help channel of steemit.chat. I have messaged you there, let's continue the conversation in chat.

**Comment by plink01001** [comment]
Closing issue, as all signup functionality has been converted to the new https://github.com/steemit/faucet.
